automation/test.sh

```diff
@@ -55,10 +46,10 @@ vagrant up --provider=libvirt
 if [ $? -ne 0 ]; then
   # After a workspace cleanup we loose our .vagrant file, this means that we have to clean up libvirt
   vagrant destroy
-  virsh undefine kubevirt_master
   virsh destroy kubevirt_master
-  virsh undefine kubevirt_node0
+  virsh undefine kubevirt_master
   virsh destroy kubevirt_node0
+  virsh undefine kubevirt_node0
```

Comment:
It is possible you may find you have to manually remove the storage volumes too here if you lost the .vagrant file.   If you've tested this with multiple runs and it works, then the volumes probably aren't an issue.